src/videojs-http-streaming.js

```diff
@@ -1079,7 +1079,14 @@ class VhsHandler extends Component {
 
     this.mediaSourceUrl_ = window.URL.createObjectURL(this.playlistController_.mediaSource);
 
-    this.tech_.src(this.mediaSourceUrl_);
+    // If we are playing HLS with MSE in Safari, add source elements for both the blob and manifest URLs.
+    // The latter will enable Airplay playback on receiver devices.
+    if ((videojs.browser.IS_ANY_SAFARI || videojs.browser.IS_IOS) && this.options_.overrideNative && this.options_.sourceType === 'hls') {
+      this.tech_.addSourceElement(this.mediaSourceUrl_);
+      this.tech_.addSourceElement(this.source_.src);
+    } else {
+      this.tech_.src(this.mediaSourceUrl_);
```

Comment:
Is there any difference between `this.tech_.addSourceElement(this.mediaSourceUrl_)` and `this.tech_.src(this.mediaSourceUrl_)`?

Could we just do one in every case, and conditionally add the manifest URL source?

Reply:
`this.tech_.src()` sets the source by setting the `src` attribute on the `<video>` element, while `this.tech_.addSourceElement()` adds the source as a separate `<source>` element that is a child of the `<video>` element. We definitely want both the Media Source and Airplay-compatible sources to be `<source>` elements when using MSE in Safari, per the Webkit doc, and my inclination was to preserve the existing behavior in all other cases (setting the `<video>`'s `src` attribute)